### ISSUE TITLE
Combining Diacritics on Same Side Can Commute 

### ISSUE THREAD

**Issue by Richard57** [w3c/charmod-norm#167]
In Section 2.2.1 'Order of combining marks', it states that 'The order of combining diacritics on the same side have a positional meaning'.  This is not true for Thai minority languages, e.g. Northern Khmer กฺู 'cow' U+0E01 THAI CHARACTER KO KAI, U+0E3A THAI CHARACTER PHINTHU, U+0E39 THAI CHARACTER SARA UU, whose combining marks have different canonical combining classes.  Additionally, the Sanskrit vowel sequences <U+0F75 TIBETAN VOWEL SIGN UU>, <U+0F71 TIBETAN VOWEL SIGN AA, U+0F74 TIBETAN VOWEL SIGN U> and <U+0F74, U+0F71> are canonically equivalent.

**Comment by aphillips** [comment]
Addressed in https://github.com/aphillips/charmod-norm/commit/e99eb2e91627e629ab090b3ce55bc16f1794c091 when I addresses Najib's similar comment about Arabic. Are you satisfied by this change?

**Comment by aphillips** [comment]
Duplicates #162

**Comment by Richard57** [comment]
The change solves the problem I raised.  There are two niggles.

Is the term 'diacritic' to be taken as equivalent to the term 'mark'?  I have recently discovered that Unicode defines the property 'diacritic' differently, so that viramas are diacritics but Indic vowel marks are not.  I can't find a W3C definition of 'diacritic'.

Additionally, "doesn't matter to the presentation" should be "doesn't matter to canonical equivalence".  When not canonically equivalent, the order of marks can matter to the reading even though it does not affect the rendering!  A simple but heterodox example is <1A56, 1A6B> /lo/ v. <1A6B, 1A56> /on/.  (I was startled to find two examples of the latter in a dictionary.)  These marks are above and below.  For marks on the same side, above in this case, I could offer <1A65, 1A7B> and <1A7B, 1A65>, though I believe the possible renderings have different frequencies for the two codepoint orders.
